tidb-in-kubernetes/aws-eks-tutorial.md

```diff
@@ -0,0 +1,187 @@
+# 在 AWS EKS 上部署 TiDB Operator 和 TiDB 集群
+
+本文档描述如何使用个人电脑 (Linux or macOS) 在 AWS EKS 上部署 TiDB Operator 和 TiDB 集群用于开发或者测试。
+
+## 部署环境
+
+部署前，请确认已安装以下软件：
+* [awscli](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) >= 1.16.73，控制 AWS 资源
+
+  要与 AWS 交互，必须 [配置](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) `awscli`。最快的方式是使用 `aws configure` 命令:
+
+  ``` shell
+  # 替换下面的 AWS Access Key ID 和 AWS Secret Access Key
+  $ aws configure
+  AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+  AWS Secret Access Key [None]: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  Default region name [None]: us-west-2
+  Default output format [None]: json
+  ```
+  > **注意：**
+  >
+  > Access key 必须至少具有以下权限：创建 VPC，创建 EBS，创建 EC2 和创建 Role。
```

Comment:
```suggestion
  > Access key 必须至少具有以下权限：创建 VPC，创建 EBS，创建 EC2 和创建 Role。
```
Note that we should add a new line **before** and **after** some special elements such as code blocks, notes, etc.